DESIGN.md

```diff
@@ -54,9 +37,7 @@ PK | series_set | As `{metric_name},{tagK}={tagV},...` with tagK sorted
 CK | ts | timestamp of ingested metric
 &nbsp; | value | Metric value as double
 
-To be a bit more explicit, all these tables are updated/inserted on ingest.  The "metric_names" table is only read by the "metricNames" metadata query; the "tag_keys" by the "tagKeys" metadata query and the "tag_values" by the "tagValues" metadata query.  The "series_set" and "data_raw" tables are used in combination by data queries and "data_raw" is also used during downsampling.
-
-Since the process of downsampling will derive new metrics, the existence of those is tracked by adding into the `aggregators` set. The query API can be updated to take into account the metric name and aggregation to decide what downsampled data, if any, can be used.
+To be a bit more explicit, all these tables are updated/inserted on ingest.  The "metric_names" table is during metadata queries for tenants and metric names. Metadata queries for tag keys and values given tenant and metric name are resolved by the "series_sets" table. The "series_set" and "data_raw" tables are used in combination by data queries and "data_raw" is also used during downsampling.
```

Comment:
The "metric_names" table is ??read?? during

Reply:
Oops, yes.